docs/get-started/best-practices/best-practices-for-harvard.md

```diff
@@ -0,0 +1,208 @@
+and [Research Data](https://security.harvard.edu/data-security-levels-research-data-examples).
+
+Additionally, if your work involved individuals situated in a European Economic
+Area, you may be subject to the requirements of the General Data Protection
+Regulations and more information about your responsibilities can be found
+[here](https://security.harvard.edu/eu-general-data-protection-regulation-gdpr).
+
+## Host Protection
+
+The primary focus of this guide is to provide you with security essentials that
+we both support and that you can implement with little effort.
+
+### Endpoint Protection
+
+Harvard University uses the endpoint protection service: **Crowdstrike**, which
+actively checks a machine for indication of malicious activity and will act to
+both block the activity and remediate the issue. This service is offered free to
+our community members and requires the installation of an agent on the server
+that runs transparently. This software enables the Harvard security team to
+review security events and act as needed.
+
+Crowdstrike can be downloaded from our repository at:
+[agents.itsec.harvard.edu](https://agents.itsec.harvard.edu/) this software is required
+for all devices owned by Harvard staff/faculty and available for all operating
+systems.
+
+!!! note "Please note"
+    To acess this repository you need to be in **Harvard Campus Network**.
+
+### Patch/Update Regularly
+
+It is common that vendors/developers will announce that they have discovered a new
+vulnerability in the software you may be using. A lot of these vulnerabilities
+are addressed by new releases that the developer issues. Keeping your software
+and server operating system up to date with current versions ensures that you are
+using a version of the software that does not have any known/published vulnerabilities.
+
+### Vulnerability Management
+
+Various software versions have historically been found to be vulnerable to specific
+attacks and exploits. The risk of running older versions of software is that you
+may be exposing your machine to a possible known method of attack.
+
+To assess which attacks you might be vulnerable to and be provided with specific
+remediation guidance, we recommend enrolling your servers with our Tenable service
+which periodically scans the software on your server and correlates the software
+information with a database of published vulnerabilities. This service will enable
+you to prioritize which component you need to upgrade or otherwise define which
+vulnerabilities you may be exposed to.
+
+The Tenable agents runs transparently and can be enabled to work according to
+the parameters set for your school; the agent can be downloaded
+[here](https://agents.itsec.harvard.edu/) and configuration support can be found by
+filing a support request via HUIT support ticketing system:
+[ServiceNow](https://harvard.service-now.com/ithelp?id=submit_ticket&sys_id=3f1dd0320a0a0b99000a53f7604a2ef9).
```

Comment:
would this link work as well?
https://harvard.service-now.com/ithelp?id=submit_ticket

Reply:
It does not work with this shorten link once Harvard login. so we need to put the longer url link as listed here.